packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskInputField/TaskInputField.js

```diff
@@ -1,69 +1,48 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import styled, { ThemeProvider } from 'styled-components'
-import theme from 'styled-theming'
+import styled, { withTheme } from 'styled-components'
 import { darken, lighten } from 'polished'
-import zooTheme from '@zooniverse/grommet-theme'
+import { Text } from 'grommet'
 
 import TaskInputLabel from './components/TaskInputLabel'
-import { pxToRem } from '@zooniverse/react-components'
 
-const DEFAULT = {
-  backgroundColor: theme('mode', {
-    dark: zooTheme.global.colors['dark-3'],
-    light: zooTheme.global.colors['light-1']
-  }),
-  border: theme('mode', {
-    dark: `2px solid ${zooTheme.global.colors.text.dark}`,
-    light: '2px solid transparent'
-  }),
-  color: theme('mode', {
-    dark: zooTheme.global.colors.text.dark,
-    light: zooTheme.global.colors.text.light
-  })
-}
-
-const HOVER = {
-  gradientTop: theme('mode', {
-    dark: darken(0.04, zooTheme.global.colors['neutral-2']),
-    light: lighten(0.05, zooTheme.global.colors['accent-2'])
-  }),
-  gradientBottom: theme('mode', {
-    dark: darken(0.11, zooTheme.global.colors['neutral-2']),
-    light: darken(0.11, zooTheme.global.colors['accent-2'])
-  }),
-  color: theme('mode', {
-    dark: zooTheme.global.colors.text.dark,
-    light: 'black'
-  })
-}
 
-const CHECKED = {
-  background: theme('mode', {
-    dark: zooTheme.global.colors.brand,
-    light: zooTheme.global.colors.brand
-  }),
-  border: theme('mode', {
-    dark: `2px solid ${zooTheme.global.colors.brand}`,
-    light: '2px solid transparent'
-  }),
-  color: theme('mode', {
-    dark: zooTheme.global.colors.text.dark,
-    light: 'white'
-  })
+function getHoverStyles (props, active = false) {
+  if (props.theme.dark) {
```

Comment:
I'd probably do some destructuring to make this easier to read:

```
const { theme: { dark, global: { colors } } } = props
// `dark` and `colors` are now available
```

Reply:
Ok I've added the destructuring.